services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -87,7 +87,7 @@ public Response getAllRealizations(@Context HttpServletRequest httpRequest,
                                      @QueryParam("offset")
                                      int offset,
                                      @Parameter(description = "Limit of result")
-                                     @DefaultValue("10")
+                                     @DefaultValue("25")
```

Comment:
There is no real default value if you already use this check here :
 ```
   if (!isXlsx && limit <= 0) {
      return Response.status(Response.Status.BAD_REQUEST).entity("Limit must be positive").build();
    }
```

@AzmiTouil @ArfaouiMohamedAmine this line has to be removed

Reply:
@AzmiTouil @boubaker  done, changes has been applied here https://github.com/Meeds-io/gamification/pull/471/files